modules/viewing-pipeline-task-executions.adoc

```diff
@@ -0,0 +1,32 @@
+:_module-type: PROCEDURE
+
+[id="viewing-pipeline-task-executions_{context}"]
+= Viewing pipeline task executions
+
+[role='_abstract']
+When a pipeline run executes, you can view details of executed tasks in each step in a pipeline run from the {productname-short} dashboard. A step forms part of a task in a pipeline.
+
+.Prerequisites
+* You have logged in to {productname-long}.
+ifndef::upstream[]
+* If you are using specialized {productname-short} groups, you are part of the user group or admin group (for example, {oai-user-group} or {oai-admin-group}) in OpenShift.
+endif::[]
+ifdef::upstream[]
+* If you are using specialized {productname-short} groups, you are part of the user group or admin group (for example, {odh-user-group} or {odh-admin-group}) in OpenShift.
+endif::[]
+* You have previously created a data science project that is available and contains a pipeline server.
+* You have imported a pipeline to an active pipeline server.
+* You have previously triggered a pipeline run.
+
+.Procedure
+. From the {productname-short} dashboard, click *Experiments* -> *Executions*.
+. Select the *project* that contains the pipeline experiment for the pipeline task executions that you want to view.
+. Study the table showing a list of pipeline task executions.
```

Comment:
Same as previous - this isn't really a UI step, but could be a good place for a validation line.  (e.g. 'A list of pipeline task executions is displayed.')